src/server/protocol.ts

```diff
@@ -2,103 +2,103 @@
  * Declaration module describing the TypeScript Server protocol
  */
 namespace ts.server.protocol {
-    export namespace CommandTypes {
-        export type Brace = "brace";
+    export enum CommandTypes {

```

Comment:
this should be `const enum`, since it does not exist at time of building the client against the protocol file.